server/Controllers/Products/delete.js

```diff
@@ -17,14 +17,14 @@ export async function deleteProduct(req, res) {
                 }
                 else{
                     const foundProduct = await Product.findOne({_id:ObjectIdProjectId})
-                    const foundFeaturedProduct = await FeaturedProduct.findOne({'productId':ObjectIdProjectId})
+                    const foundFeaturedProduct = await FeaturedProduct.findOne({'product':ObjectIdProjectId})
```

Comment:
From the readability aspects - its better to name it as `productID` instead of `product`

Reply:
Sure we'll change it